llama_index/llama_dataset/download.py

```diff
@@ -14,8 +17,10 @@
 def download_llama_dataset(
```

Comment:
this seems a light wrapper around download_llama_dataset in `download.dataset`, can we consolidate? e.g. just move the logic from there to here

Reply:
good call! i'll consolidate as you suggest

Reply:
I think if we do this, then we do the same `llama_module` and others, as they follow a similar light-wrapper pattern.

Reply:
maybe, but in those cases it's packs/tools/datasets sharing the same `download_llama_module` whereas here it's just one function on top of another

Reply:
ahh right. Okay I'd be good with leaving those as wrappers.

Reply:
@jerryjliu: latest update on this. I just resolved some conflicts that resulted in our additional `track_downloads`, and doing this made me realize that this light wrapper in a way provides a similar interface for `download_llama_dataset()` to the other llama-hub download methods:

- `download_llama_pack()`
- `download_tool()`
- `download_loader()`

In that sense, may be its good to keep the light wrapper here? Thoughts?